A16-binary-logging.md

```diff
@@ -60,9 +63,9 @@ message GrpcLogEntry {
   // definition, but the same meaning is expressed here.
   enum EventType {
     EVENT_TYPE_UNKNOWN = 0;
-    // Headers sent from client to server
+    // Header sent from client to server
     EVENT_TYPE_CLIENT_HEADER = 1;
```

Comment:
Note that the header includes things that are not "metadata" (e.g. the method name).  Does that influence your opinion on this?

Reply:
That doesn't really bother me.  Those other fields are associated with these ops.  The names of the ops are IMHO the important thing here, and I think it's important for us to be consistent.

Reply:
I don't think it is appropriate to call this initial_metadata, as it would _contain_ initial_metadata. (Calling what it contains just "metadata" is fine because there is context)

Reply:
But the event is the client sending initial metadata.  I thought we had agreed to use that term in the call semantics doc?  If so, why would we use something different here?

Reply:
The [semantics PR as it is currently written](https://github.com/grpc/grpc/pull/15460/files#diff-449981901227883dfbe29af4b096d0daR238) and [wire protocol doc](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md) use the terms "request headers", "response headers", and "trailers". Aren't these the ops we are trying to be consistent with?

Reply:
@ejona86 and @dfawley and I have discussed this and decided to try the terms "gRPC header" and "gRPC trailer" (both singular).  So it looks like this PR is using the right terms.